tests/components/mqtt/test_light_json.py

```diff
@@ -1338,7 +646,7 @@ async def test_sending_mqtt_commands_and_optimistic(
         }
     ],
 )
-async def test_sending_mqtt_commands_and_optimistic2(
+async def test_sending_mqtt_commands_and_optimistic(
```

Comment:
Renamed because the old `test_sending_mqtt_commands_and_optimistic` test used the removed legacy `color_mode` support.